config/locales/es.yml

```diff
@@ -1815,33 +1815,33 @@ es:
           title: Edita tu información básica
         prior_tax_year_agi:
           edit:
-            help_text: Puede encontrar esta información en la línea 11 de su declaración de impuestos del %{prior_tax_year}. Introduzca el monto correcto de su declaración de %{prior_tax_year} o su declaración será rechazada.
-            label: "%{prior_tax_year} AGI"
-            title: Edite el ingreso bruto ajustado de %{prior_tax_year}
+            help_text: Puede encontrar esta información en la línea 11 de su declaración de impuestos %{prior_tax_year}. Introduzca la cantidad correcta de su declaración del %{prior_tax_year} o su declaración será rechazada.
+            label: 'El Ingreso Bruto Ajustado para %{prior_tax_year} en inglés: AGI'
+            title: 'Edite su Ingreso Bruto Ajustado, en inglés: "Adjusted Gross Income" para %{prior_tax_year}'
         spouse:
           help_text: Confirmar / editar la información básica de su cónyuge
           title: Editar la información básica de su cónyuge
         spouse_prior_tax_year_agi:
           edit:
-            help_text: Puede encontrar esta información en la línea 11 de la declaración de impuestos del %{prior_tax_year} de su cónyuge. Introduzca la cantidad correcta o su declaración será rechazada.
-            label: AGI de %{prior_tax_year} del cónyuge
-            title: Edite el ingreso bruto ajustado de %{prior_tax_year} de su cónyuge
+            help_text: Puede encontrar esta información en la línea 11 de la declaración de impuestos de su cónyuge del %{prior_tax_year}. Introduzca la cantidad correcta o su declaración será rechazada.
+            label: 'El Ingreso Bruto Ajustado, en inglés: AGI del cónyuge para %{prior_tax_year}'
+            title: Edite los Ingresos Brutos Ajustados de su cónyuge de %{prior_tax_year}
         verification:
-          i_dont_have_id: No tengo identificación
-          id: Su licencia de conducir o tarjeta de identificación estatal
-          id_label: Foto de su identificación
-          info: 'Para proteger su identidad y verificar su información, necesitaremos que envíe las dos fotos siguientes:'
+          i_dont_have_id: No tengo identificación, o ID
+          id: Su licencia de manejar o tarjeta de identificación del estado, o ID
+          id_label: Identificación con foto
```

Comment:
The English here is: "Photo of your ID". I am worried that the Spanish is wrong. I'll merge it, but FYI.